onnxruntime/core/providers/cpu/controlflow/if.cc

```diff
@@ -104,12 +104,24 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(If,
                                    If);
 
 // float 8 support was added.
-ONNX_CPU_OPERATOR_KERNEL(If,
-                         19,
-                         KernelDefBuilder()
-                             .TypeConstraint("B", DataTypeImpl::GetTensorType<bool>())
-                             .TypeConstraint("V", DataTypeImpl::AllTensorAndSequenceTensorAndOptionalTypesIRv9()),
-                         If);
+ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
+    If,
+    19,
+    20,
+    KernelDefBuilder()
+        .TypeConstraint("B", DataTypeImpl::GetTensorType<bool>())
+        .TypeConstraint("V", DataTypeImpl::AllTensorAndSequenceTensorAndOptionalTypesIRv9()),
+    If);
+
+// uint4 and int4 support was added.
+// TODO(adrianlizarraga): Implement int4 and uint4 support.
```

Comment:
I assume no changes specific to these control-flow ops will be required? It seems to me that the support required is probably the generic tensor implementation for int4/uint4 (with operations to copy such tensors etc.)?

Reply:
That's right, we would need to add the int4/uint4 tensor implementation.